src/ui/Views/Legal/Cookies.cshtml

```diff
@@ -91,7 +91,7 @@
         <h3 class="heading-medium">
             Other cookies
         </h3>
-        <p>‘Publish postgraduate teacher training courses’ may include:</p>
+        <p>Other ‘Publish postgraduate teacher training courses’ may include:</p>
```

Comment:
`Other ‘Publish postgraduate teacher training courses’ may include:`
should be
`Other ‘Publish postgraduate teacher training courses’ cookies may include:`

Reply:
I need a cup of now tea